src/styles/navigation.scss

```
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    &__item {
+      margin: 0 10px;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+
+    font-weight: 500;
+    line-height: 14px;
+    color: #000;
```

Create a variable for this color & Nav links color is not black any more\

```suggestion
    color: #060b35;
```